Sources/ImageCramCLI/ImageCramCommand.swift

```diff
@@ -15,10 +15,10 @@ struct ImageCramCommand: ParsableCommand {
     var output: String?
 
     @Flag(name: .shortAndLong, help: "Output the version number")
-    var version: Bool
+    var version: Bool = false
```

Comment:
This change is needed as per Swift Argument Parser 0.2.0 and later, see changelog [here](https://github.com/apple/swift-argument-parser/blob/e1465042f195f374b94f915ba8ca49de24300a0d/CHANGELOG.md#020---2020-06-23).